polkadot/node/core/pvf/src/testing.rs

```diff
@@ -117,7 +117,11 @@ pub fn build_workers_and_get_paths(is_bench: bool) -> (PathBuf, PathBuf) {
 			}
 		}
 
-		build_workers(is_bench);
+		// release build
+		#[cfg(not(debug_assertions))]
+		let release_build = true;
```

Comment:
This is not the correct check.

https://github.com/paritytech/polkadot-sdk/blob/84ddbaf68445fed23a88a086ee37bdcdbcb7356a/substrate/client/consensus/slots/build.rs#L21-L25

with

https://github.com/paritytech/polkadot-sdk/blob/84ddbaf68445fed23a88a086ee37bdcdbcb7356a/substrate/client/consensus/slots/src/lib.rs#L230 (using `release`).

is the proper approach.

Reply:
You could also just forward `profile` to the build using `--profile {profile}`

Reply:
Nice! I tried tapping PROFILE in the code, not present obviously. Didn't occur to me I should try the build script. Curious, would `#[cfg(not(debug_assertions))]` not work as expected in some case or is it just too hacky?

Reply:
We are for example enable `debug_assertions` in release mode for CI. But your code would assume that it is running in `Debug`.

Reply:
Didn't know we could that. :)